README.md

```diff
@@ -1 +1,94 @@
-# Chrome Enamel STEP interns project repository
+# Chrome Enamel STEP interns project repository :elephant:
+
+Dashboard for HTTP-related stuff to deprecate in the future by Chrome team
```

Comment:
nit: Maybe pick a word more formal than "stuff"? Maybe something like "Dashboard for HTTP-related support that are targets for future deprecation in Chrome." That does still sound a bit awkward, so feel free to suggest something else.

Reply:
I don't have any other suggestion since you probably have more clue on the final purpose of the dashboard, so I'm good with `support`.

 I replaced every occurrence of `stuff`